tablo/geom_utils.py

```diff
@@ -6,9 +6,11 @@
 from math import sqrt, fabs, cos, radians
 from itertools import product
 
-
 GLOBAL_EXTENT_WEB_MERCATOR = (-20037508.342789244, -20037342.166152496, 20037508.342789244, 20037342.16615247)
 SQL_BOX_REGEX = re.compile('BOX\((.*) (.*),(.*) (.*)\)')
+WKT_POINT_REGEX = re.compile('POINT\((.*) (.*)\)')
```

Comment:
I was considering points as one dimensional entities and polygon and polylines as 2d stuff and that's the reason behind my weird naming here; though I just dropped `WKT_POINT_REGEX` and modified the code so all geometries use one regex and renamed the patterns and hopefully they are more meaningful now.